src/admin/pages/users/usersInfoGrid.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+    <div>
+        <b-container fluid class="container-main">
+            <b-row>
+                <b-col lg="2" md="9" sm="9" class="col-xs-12 col-sm-6 col-md-3 example-col">
+                    <b-input-group size="md" class="mb-3 search-box " >
+                        <b-form-input id="search-textbox" placeholder="Search users..." v-model="searchUserText" @keyup.native="userGridSearch()"></b-form-input>
+                    </b-input-group>
+                </b-col>
+            </b-row>
+        </b-container>
+        <div id="vueapp" class="vue-app" >
+            <kendo-grid
+                :height="500"
+                :data-source="users"
+                :resizable="true"
+                :filterable="true"
+                :sortable-allow-unsort="true"
+                :sortable-show-indexes="true"
+                :scrollable-virtual="true"
+                :pageable-numeric="false"
+                :pageable-previous-next="false"
+                :pageable-messages-display="'Showing {2} users'"
+                :editable="'popup'"
+                :toolbar="[{name: 'excel', text: 'Excel'}]"
+                :excel-file-name="'Motadata_UserListing.xlsx'"
+                :excel-filterable="true"
+            >
+                <kendo-grid-column :selectable="true" :width="35"></kendo-grid-column>
+                <kendo-grid-column :field="'UserName'"
+                                   :template="this.toggleTemplate()"
+                                   :width="180"></kendo-grid-column>
+                <kendo-grid-column :field="'Role'"
+                                   :width="140"></kendo-grid-column>
+                <kendo-grid-column :field="'AssignedGroups'"
+                                   :width="180" ></kendo-grid-column>
+                <kendo-grid-column :field="'Email'"
+                                   :width="220" ></kendo-grid-column>
+                <kendo-grid-column :field="'Description'"
+                                   :width="220" ></kendo-grid-column>
+
+                <kendo-grid-column :text="'Actions'"
+                                   :command="['edit', 'destroy']"
+                                   :title="'Actions'"
+                                   :width="150"></kendo-grid-column>
+            </kendo-grid>
+        </div>
+    </div>
+
+</template>
+
+<script>
+    import JsZip from 'jszip';
```

Comment:
Why do you need JsZip?

Reply:
JsZip used for kendo grid export functionality.
If i don't used window.JSZip = JsZip;  then export functionality don't work.

Reply:
okay. We will later try to bind it to local scope.